scripts/buildUserscript.ts

```diff
@@ -8,6 +8,7 @@ import packageJson from '../package.json';
 import config, { PORT_ENV_KEY } from '../src/config';
 
 const script = packageJson.userscript;
+const localCoreName = 'core';
```

Comment:
I think, we also have to pass this as parameter to Terser

Reply:
Fixed in [d2c9831](https://github.com/LSS-Manager/LSSM-V.4/pull/2865/commits/d2c98310f1fb6c4e28c81efd0d0fda5a91e8487a)